WordPress/src/main/res/color/grey_min_orange_jazzy_grey_disabled_selector.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yeah, I agree.  I named it _what it is_ rather than _what is it for_ so that we can reuse it elsewhere without confusion.  If I named it `reader_button_like` and wanted to use it for the comment approve button, it wouldn't make as much sense.  Also, it's easy to know what colors are used by just reading the name.  If I named it `reader_button_like`, there's no way of knowing what colors that selector uses from the name.  I'm open to using a better naming convention if we can think of one.